app/src/main/java/com/yeonae/chamelezone/view/mypage/myplace/PlaceRegisterActivity.kt

```diff
@@ -146,9 +163,20 @@ class PlaceRegisterActivity : AppCompatActivity(), PlaceContract.View,
 
     override fun onActivityResult(requestCode: Int, resultCode: Int, data: Intent?) {
         super.onActivityResult(requestCode, resultCode, data)
-        if (requestCode == REQUEST_CODE) {
+        if (requestCode == ADDRESS_REQUEST_CODE) {
+            if (resultCode == RESULT_OK) {
+                tv_place_address.text = data?.getStringExtra(RESULT)
+            }
+        } else if (requestCode == OPENING_HOURS_REQUEST_CODE) {
             if (resultCode == RESULT_OK) {
-                tv_place_address.text = data?.getStringExtra("result")
+                openingTime = data?.getStringArrayListExtra(OPENING_HOURS) as ArrayList<String>
+                Log.d("openingTime", openingTime.toString())
+                val openingHours =
+                    openingTime.toString().replace(", ", "\n")
+                        .replace("[", "")
+                        .replace("]", "")
```

Comment:
```suggestion
                val openingHours =
                    openingTime.joinToString("\n")
```
어레이로 받아왔는데 다시 그거를 잘라서쓴다??🤔

Reply:
수정했습니다